resource-watch/resource-watch
# Move topics to dashboards page + dashboards page initial updates

## Overview
This PR:
 - adds new /dashboards page with content like on 'topics page'. (we still use topics components. They will be renamed in future)
 - adds new routes /dashboards/:id are equal to /topics/:id
 - updates routes from /topics to /dashboards
 - adds "more" option to header-dropdown menu
 - adds new Hero text and new section texts,
 - adds gallery prototype to /dashboards from https://resourcewatch.org/data/dashboards (Visuals will update in future PR)
 - removed /data/dashboards route from the project. /data/dashboards/:slug for custom dashboards work.
## Testing instructions
Use staging API. Go to http://localhost:9000/dashboards or click dashboards on the header-menu.
Check all texts on this page.
Click on featured dashboard item to check /dashboards/:id page.
Click on gallery item or follow http://localhost:9000/data/dashboards/testing-2-c0f6e3a1-b4e3-4999-aec2-49137e0cd2ae
Follow http://localhost:9000/data/dashboards to get 404 page (removed)

## Pivotal tasks
[Menu title and routes](https://www.pivotaltracker.com/n/projects/1374154/stories/169732634)
[Hero text and first section text](https://www.pivotaltracker.com/n/projects/1374154/stories/169732737)
[More oprion](https://www.pivotaltracker.com/n/projects/1374154/stories/169732869)
[Remove unnecessary route](https://www.pivotaltracker.com/n/projects/1374154/stories/169733032)
[Add new "Dashboards gallery" section on the "Dashboards" page.](https://www.pivotaltracker.com/n/projects/1374154/stories/169732990)

---

## Checklist before submitting
